routing/endpointregistry.go

```diff
@@ -166,6 +168,9 @@ func (r *EndpointRegistry) updateStats() {
 			requests := e.totalRequests[curSlot].Load()
 			if requests > r.minRequests {
 				failedRoundTripsRatio := float64(failed) / float64(requests)
+				if failedRoundTripsRatio > 0.0 {
+					log.Infof("passive health check: marking %q as unhealthy due to failed round trips ratio: %0.2f", key, failedRoundTripsRatio)
```

Comment:
Log messages should start from capital, error messages from small letter.

Reply:
Changed both log messages.